tests/epyccel/test_epyccel_optional_args.py

```diff
@@ -8,7 +8,6 @@
 #------------------------------------------------------------------------------
 @pytest.mark.parametrize( 'language', [
         pytest.param("fortran", marks = [
-            pytest.mark.xfail(reason="f2py does not support optional arguments"),
             pytest.mark.fortran]),

```

Comment:
I have the same comment as above.
BTW, why do we need to pass the same pytest marks to so many functions in this module?

Reply:
We mark functions depending on which language they are translated to. This is actually done in every file, but when there is no language specific xfail mark, it can be handled with the fixture so it is invisible everywhere else